src/pages/CircleAdminPage/AdminIntegrations.tsx

```diff
@@ -59,8 +59,14 @@ export const AdminIntegrations = ({ circleId }: { circleId: number }) => {
   }, [integrations.refetch, deleteIntegration]);
 
   return (
-    <div style={{ display: 'grid' }}>
-      <p className={classes.subTitle}>Integrations</p>
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+      }}
+    >
+      <Text className={classes.subTitle}>DeWork Integration</Text>
```

Comment:
```suggestion
      <Text className={classes.subTitle}>Connected Integrations</Text>
```

Other integrations may show up in this section in the future

Reply:
done